scripts/rewrite/rewrite.js

```diff
@@ -0,0 +1,207 @@
+// ecrire une doc
+import fsApi from 'fs';
+import path from 'path';
+import { pathToFileURL, fileURLToPath } from 'url';
+
+import config from 'config';
+import mime from 'mime';
+
+import Git from '../../src/app/history/git.js';
+
+const fs = fsApi.promises;
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+export const SNAPSHOTS_SOURCE_PATH = path.resolve(__dirname, '../../', config.get('rewrite.snapshotsSourcePath'));
+export const SNAPSHOTS_TARGET_PATH = path.resolve(__dirname, '../../', config.get('history.snapshotsPath'));
+
+const README_PATH = path.resolve(__dirname, './initFiles/snapshot-readme.md');
+const LICENSE_PATH = path.resolve(__dirname, './initFiles/snapshot-license');
+
+const EMPTY_TOS_BACK_CONTENT = `<!DOCTYPE html><html><head></head><body>
+
```

Comment:
Need an empty line?

Reply:
Yep, to be the exact same content of ToSBack empty documents